keps/sig-network/20191014-nodelocaldns-GA-proposal.md

```diff
@@ -0,0 +1,103 @@
+---
+title: NodeLocal DNSCache GA Proposal
```

Comment:
I don't think we should create a separate KEP for GA-ing the feature.
We should reflect the thoughts we have in the existing nodelocal-cache-dns KEP:
https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/0030-nodelocal-dns-cache.md

Reply:
Makes sense -- pavithra, maybe amend that doc instead?